src/accsr/remote_storage.py

```diff
@@ -498,12 +498,12 @@ def _execute_sync_from_summary(
             )
 
         with tqdm(total=summary.size_files_to_sync(), desc="Progress (Bytes)") as pbar:
-            for synced_object in summary.files_to_sync:
-                sync_obj = self.execute_sync(
-                    synced_object, direction=summary.sync_direction, force=force
+            for sync_obj in summary.files_to_sync:
+                synced_obj = self.execute_sync(
+                    sync_obj, direction=summary.sync_direction, force=force
                 )
-                pbar.update(synced_object.local_size)
```

Comment:
Previously, the `pbar` was updated with the objects from `summary.files_to_sync` which had `.local_size` (for my case at least). The downloaded objects has the attribute filled. 
I think the naming makes more sense this way.

Reply:
Hi Basti. I haven't received notifications from github and haven't had a look at accsr in too long. Sry for ignoring your MRs, I really wasn't aware of them. Merging now (and I will also fix the pipeline and add tests)